src/OpenTelemetry.Instrumentation.AspNetCore/README.md

```diff
@@ -311,6 +311,52 @@ the library you can do so as follows:
 });
 ```
 
+#### Enriching ASP.NET Core Request Metrics
+
+ASP.NET Core supports enriching request metrics using `IHttpMetricsTagsFeature`.
```

Comment:
please call out that this is only applicable for .NET 8 and newer.

Reply:
Fixed in 99a4211340adb3c944e2380f9f010a72d6933dac